client/wfnews-war/src/main/angular/src/app/components/wildfires-list-header/bans-list/bans-list.component.mobile.ts

```diff
@@ -9,7 +9,7 @@ import { ResourcesRoutes } from '@app/utils';
 })
 export class BansListComponentMobile extends BansListComponent {
   navigateToFullDetails(item: any) {
-    if (item.id) {
+    if (item.id !== undefined && item.id !== null) {
```

Comment:
Doesn't this change do the same thing as the original code?

Reply:
this will exclude the situation when Id =0